pss/notify/notify_test.go

```diff
@@ -16,17 +16,15 @@ import (
 	"github.com/ethereum/go-ethereum/p2p/enode"
 	"github.com/ethereum/go-ethereum/p2p/simulations"
 	"github.com/ethereum/go-ethereum/p2p/simulations/adapters"
-	whisper "github.com/ethereum/go-ethereum/whisper/whisperv6"
 	"github.com/ethersphere/swarm/network"
 	"github.com/ethersphere/swarm/pss"
 	"github.com/ethersphere/swarm/state"
 )
 
 var (
-	loglevel = flag.Int("l", 3, "loglevel")
-	psses    map[string]*pss.Pss
-	w        *whisper.Whisper
-	wapi     *whisper.PublicWhisperAPI
+	loglevel      = flag.Int("l", 3, "loglevel")
+	psses         map[string]*pss.Pss
+	cryptoBackend pss.CryptoBackend
```

Comment:
just `crypto`

Reply:
We are importing "github.com/ethereum/go-ethereum/crypto" in the package. Maybe we should encapsulate also all uses of "github.com/ethereum/go-ethereum/crypto" into our cryptoBackend also.

Reply:
Changed in commit 8fcc65a.